adam-cli/src/test/scala/org/bdgenomics/adam/cli/TransformFeaturesSuite.scala

```diff
@@ -44,9 +42,7 @@ class TransformFeaturesSuite extends ADAMFunSuite {
     val features2Adam = new TransformFeatures(args)
     features2Adam.run(sc)
 
-    val schema = Projection(featureId, contigName, start, strand)
-    val lister = new ParquetLister[Feature](Some(schema))
-    val converted = lister.materialize(outputPath).toSeq
+    val converted = sc.loadFeatures(outputPath).rdd.collect
```

Comment:
The direct schema conversion no longer works because of the attributes

Reply:
Seems reasonable to me